modules/helm-understanding-helm.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/getting-started-with-helm-on-openshift.adoc
+
+[id='understanding-helm_{context}']
```

Comment:
Should be double quotes, not single quotes. Another place in the PR does this too.